tools/generators/legacy/src/Extensions/XcodeScheme+Extensions.swift

```diff
@@ -63,39 +65,48 @@ add it or a target that depends on it to \(runnerLabel)'s `top_level_targets` at
         targets: [TargetID: Target],
         labelTargetInfos: [BazelLabel: XcodeScheme.LabelTargetInfo],
         allBazelLabels: Set<BazelLabel>,
-        runnerLabel: BazelLabel
+        runnerLabel: BazelLabel,
+        aliasLabels: [BazelLabel: [BazelLabel]]
     ) throws -> [BazelLabel: TargetID] {
         var resolvedTargetIDs: [BazelLabel: TargetID] = [:]
 
         // Identify the top-level targets
         var topLevelLabels: Set<BazelLabel> = []
         var topLevelTargetIDs: Set<TargetID> = []
         var topLevelPlatforms: Set<Platform> = []
-        for label in allBazelLabels {
-            let labelTargetInfo = try labelTargetInfos.value(
-                for: label,
-                message: aliasErrorMessage(
-                    runnerLabel: runnerLabel,
-                    missingLabel: label
-                )
-            )
-            guard labelTargetInfo.isTopLevel else {
-                continue
+
+        for targetLabel in allBazelLabels {
+            var labelsToProcess: [BazelLabel] = [targetLabel]
+            if let actualLabels = aliasLabels[targetLabel], actualLabels.count > 0 {
+                labelsToProcess = actualLabels
```

Comment:
Same as the other similar code path with similar logic, we need to expand and run this for each actual label